.editorconfig

```diff
@@ -6,3 +6,31 @@ root = true
 # as some compilers complain about files not ending in newline
 [*]
 insert_final_newline = true
+
+#trim_trailing_whitespace = true
+
+# Enforce consistent indent style
+
+[*.c]
+indent_style = space
+
+[*.cc]
+indent_style = space
+
+[*.cpp]
+indent_style = space
+
+[*.h]
+indent_style = space
+
+[*.hpp]
```

Comment:
These can be combined into one section with
```editorconfig
[*.{c,cc,cpp,h,hpp}]
indent_style = space
```

Reply:
„Available since EditorConfig Core 0.11.0“
I wanted to stay backward compatible.
On the other hand, 0.11.0 is really old now.